proxy/http2/unit_tests/test_HpackIndexingTable.cc

```diff
@@ -71,7 +71,10 @@ TEST_CASE("HPACK low level APIs", "[hpack]")
       HpackIndexingTable indexing_table(4096);
 
       for (const auto &i : indexed_test_case) {
-        std::unique_ptr<HTTPHdr> headers(new HTTPHdr);
+        std::unique_ptr<HTTPHdr, void (*)(HTTPHdr *)> headers(new HTTPHdr, [](HTTPHdr *hdr) {
+          hdr->destroy();
+          delete hdr;
```

Comment:
It looks like the destructor of `HTTPHdr` should call  `destroy()`, but it triggers a crash that the same with #10541.